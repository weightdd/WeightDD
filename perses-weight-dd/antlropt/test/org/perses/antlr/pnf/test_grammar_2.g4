grammar test_grammar_2;

id
  : 'hello' 'world'
  ;

start
  : id
  | start '[]'
  | start '()'
  | start '.' id
  | start '->' id
  | start '++'
  | start '--'
  | '(' id ')'
  | 'extension' '(' id ')'
  ;
