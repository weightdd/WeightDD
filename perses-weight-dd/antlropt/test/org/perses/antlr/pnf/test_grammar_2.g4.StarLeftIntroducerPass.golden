// Auto-generated by Perses AST.
// DO NOT MODIFY.
grammar test_grammar_2;



id
    : 'hello' 'world'
    ;

aux_rule__start_2
    : '[]'
    | '()'
    | '.' id
    | '->' id
    | '++'
    | '--'
    ;

kleene_star__start_1
    : aux_rule__start_2*
    ;

aux_rule__start_3
    : id
    | '(' id ')'
    | 'extension' '(' id ')'
    ;

start
    : aux_rule__start_3 kleene_star__start_1
    ;

